pkg/workloads/console/acceptance/acceptance.go

```diff
@@ -57,23 +58,18 @@ func (r *Runner) Prepare(logger kitlog.Logger, config *rest.Config) error {
 
 func (r *Runner) Run(logger kitlog.Logger, config *rest.Config) {
 	Describe("Consoles", func() {
-		var client clientset
-
-		BeforeEach(func() {
```

Comment:
Why make this change? Was it to improve performance?

I don't think we should be just dropping the `BeforeEach` block.

As per the Ginkgo docs:
> In general, the only code within a container block should be an It block or a BeforeEach/JustBeforeEach/JustAfterEach/AfterEach block, or closure variable declarations. It is generally a mistake to make an assertion in a container block.

Maybe we want to put this stuff in a `BeforeSuite` instead?

Reply:
Going to revert this as waiting for the mutating webhook configurations while only needed to be done once can just be done in the before each without issue to simply things as we want a new client for each test.